ale_linters/scala/scalastyle.vim

```diff
@@ -0,0 +1,46 @@
+" Author: Kevin Kays - https://github.com/okkays
+" Description: Support for the scalastyle checker.
+
+let g:ale_scala_scalastyle_options =
+\   get(g:, 'ale_scala_scalastyle_options', '-c scalastyle-config.xml')
+
+function! ale_linters#scala#scalastyle#Handle(buffer, lines) abort
+    " Matches patterns line the following:
+    "
+    " warning file=/home/blurble/Doop.scala message=Missing or badly formed ScalaDoc: Extra @param foobles line=190
+
+    let l:patterns = [
+        \ '^\(.\+\) .\+ message=\(.\+\) line=\(\d\+\)$',
+        \ '^\(.\+\) .\+ message=\(.\+\) line=\(\d\+\) column=\(\d\+\)$',
+        \]
```

Comment:
Align the continued lines on the same level of indentation, but up-indent the lines inside of the List. Have a look through the codebase for examples.